java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java

```diff
@@ -150,23 +154,81 @@ public static ObjectHandle createEurostagTutorialExample1Network(IsolateThread t
         });
     }
 
+    static ArrayPointer<CCharPointerPointer> createCharPtrArray(List<String> stringList) {
+        CCharPointerPointer stringListPtr = UnmanagedMemory.calloc(stringList.size() * SizeOf.get(CCharPointerPointer.class));
+        for (int i = 0; i < stringList.size(); i++) {
+            stringListPtr.addressOf(i).write(CTypeUtil.toCharPtr(stringList.get(i)));
+        }
+        return allocArrayPointer(stringListPtr, stringList.size());
+    }
+
+    @CEntryPoint(name = "getNetworkImportFormats")
+    public static ArrayPointer<CCharPointerPointer> getNetworkImportFormats(IsolateThread thread, ExceptionHandlerPointer exceptionHandlerPtr) {
+        return doCatch(exceptionHandlerPtr, () -> createCharPtrArray(new ArrayList<>(Importers.getFormats())));
+    }
+
+    @CEntryPoint(name = "getNetworkExportFormats")
+    public static ArrayPointer<CCharPointerPointer> getNetworkExportFormats(IsolateThread thread, ExceptionHandlerPointer exceptionHandlerPtr) {
+        return doCatch(exceptionHandlerPtr, () -> createCharPtrArray(new ArrayList<>(Exporters.getFormats())));
+    }
+
+    @CEntryPoint(name = "freeNetworkFormats")
+    public static void freeNetworkFormats(IsolateThread thread, ArrayPointer<CCharPointerPointer> formatsArrayPtr) {
```

Comment:
We should probably name it with a more generic "freeStringArray" or something like that

Reply:
Done